.github/workflows/deploy-gcp-tests.yml

```diff
@@ -104,12 +104,12 @@ env:
   CACHED_STATE_UPDATE_LIMIT: 576
 
 jobs:
-  # set up the test, if it doesn't use any cached state
+  # set up and launch the test, if it doesn't use any cached state
   # each test runs one of the *-with/without-cached-state job series, and skips the other
-  setup-without-cached-state:
-    name: Setup ${{ inputs.test_id }} test
+  launch-without-cached-state:
+    name: Launch ${{ inputs.test_id }} test
     if: ${{ !inputs.needs_zebra_state }}
-    runs-on: ubuntu-latest
+    runs-on: zfnd-runners
```

Comment:
Was hoping to have an inline ignore for this one, but that's pending in `actionlint`, but
- https://github.com/rhysd/actionlint/issues/237 😩
- https://github.com/reviewdog/action-actionlint/issues/80  

This would require adding a new file to our repo (I'm not keen on the idea)

Reply:
No worries, it's just a lint.